scripts/benchmarks/run_benchmarks.sh

```diff
@@ -16,28 +16,28 @@ source ./scripts/benchmarks/configuration.sh
 
 # Standalone benchmarks
 
-cargo build --release --features=runtime-benchmarks --bin=zeitgeist
+cargo build --profile=production --features=runtime-benchmarks --bin=zeitgeist
```

Comment:
would not this increase compile time, if I just want to run benchmarks on local machine? 
Is it possible to make this override with an simple environment variable ?

Reply:
When you use `run_benchmarks.sh` it has a longer compilation time now, but with more optimizations for the benchmarks. For what purpose do you want to run the benchmarks on your local machine? Just to check if they are running? I use this script for it: 
```bash
./target/release/zeitgeist benchmark pallet --chain dev --execution=native --wasm-execution=compiled --heap-pages=4096 --pallet "*" --extrinsic "*" --steps 1 --repeat 0 --detailed-log-output
```

Reply:
sure I can use these commands directly.

Reply:
Actually I think Viveks suggestion is good. Having an ENV variable that is set to `production` by default is useful.

Reply:
AFAIK this approach will always override the variable. The correct approach in bash to use an ENV variable or a default value in case it is not set is as follows: `PROFILE="${PROFILE-production}"`

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/789/commits/84f78ae280125da3cfc97bfff149e0ceacceb047